tck/src/main/resources/jdo/datastoreidentity/org/apache/jdo/tck/pc/company/package.jdo

```diff
@@ -241,6 +244,9 @@ has datastore identity.
             <property name="phoneNumbers" persistence-modifier="persistent">
                 <map key-type="String" value-type="String"/>
             </property>
+            <property name="languages" persistence-modifier="persistent">
+                <collection type="String"/>
```

Comment:
Shouldn't that be `<collection element-type="String"/>`?

Reply:
Good catch!